packages/manager/core/generator/app/conditional-templates/dashboard/dashboard.test.ts.hbs

```diff
@@ -1,5 +1,5 @@
 import { test, expect } from '@playwright/test';
-import '../../../../../playwright/login';
+import '@/../../playwright/login';
```

Comment:
```suggestion
import '@playwright-helpers/login';
```

We should add a path `@playwright-helpers` in the tsconfig instead
`@/` is an alias for the `src/` folder